guides/common/modules/snip_prerequisite-rhel-repositories.adoc

```diff
@@ -0,0 +1,4 @@
+* You have enabled and synced the operating system repositories to {Project}, and enabled them on the hosts:
+** {RHEL} 9 for x86_64 - BaseOS and Appstream (RPMs) {endash} `rhel-9-for-x86_64-baseos-rpms` and `rhel-9-for-x86_64-appstream-rpms`
+** {RHEL} 8 for x86_64 - BaseOS and Appstream (RPMs) {endash} `rhel-8-for-x86_64-baseos-rpms` and `rhel-8-for-x86_64-appstream-rpms`
+** {RHEL} 7 Server and Extras (RPMs) {endash} `rhel-7-server-rpms` and `rhel-7-server-extras-rpms`
```

Comment:
```suggestion
ifdef::client-content-dnf[]
* You have enabled and synced the operating system repositories to {Project}, and enabled them on the hosts:
** {RHEL} 9 for x86_64 - BaseOS and Appstream (RPMs) {endash} `rhel-9-for-x86_64-baseos-rpms` and `rhel-9-for-x86_64-appstream-rpms`
** {RHEL} 8 for x86_64 - BaseOS and Appstream (RPMs) {endash} `rhel-8-for-x86_64-baseos-rpms` and `rhel-8-for-x86_64-appstream-rpms`
** {RHEL} 7 Server and Extras (RPMs) {endash} `rhel-7-server-rpms` and `rhel-7-server-extras-rpms`
endif::[]
```

This would be a nice place for a "client-os" attribute that can be set to "RHEL" so we could use `ifeval::["{client-os}" == "Red Hat Enterprise Linux"]` cc @asteflova 

My suggestion IMHO works for now without changing anything for any build target in upstream.